webplatformcompat/viewsets.py

```diff
@@ -223,7 +246,7 @@ def include_child_pages(self):
         request = getattr(self, 'request')
         child_pages = request and request.query_params.get('child_pages')
```

Comment:
Assuming that `request.query_params` is a `QueryDict`, then `child_pages` could be `None` here which won't have a `.lower()` method. Maybe have the default be `'no'`?